usaspending_api/accounts/v2/filters/account_download.py

```diff
@@ -320,12 +320,6 @@ def generate_federal_account_query(queryset, account_type, tas_id, filters):
         "submission_period": get_fyp_or_q_notation("submission"),
         "last_modified_date"
         + NAMING_CONFLICT_DISCRIMINATOR: Cast(Max("submission__published_date"), output_field=DateField()),
-        "gross_outlay_amount": Sum(
```

Comment:
When these columns where here and not in the `all_summed_cols` variable, they were being included in the GROUP BY `queryset = queryset.values(*grouped_cols)`. This was problematic because it was preventing us from getting the data down to the expected grain, federal_account_symbol and submission_period.